codes/quantum/qubits/xzzx.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
```

Comment:
@mgullans Can you help with this one? It's a 2 pointer, so should be reasonably thorough.

Reply:
Ok sure

Reply:
Needs substantial work.  There is a lot in the XZZX paper that is of interest for the zoo.